pom.xml

```diff
@@ -120,7 +120,7 @@
         <version.mysql.driver>8.0.27</version.mysql.driver>
         <version.mysql.binlog>0.25.4</version.mysql.binlog>
         <version.mongo.driver>4.3.3</version.mongo.driver>
-        <version.sqlserver.driver>7.2.2.jre8</version.sqlserver.driver>
+        <version.sqlserver.driver>9.4.1.jre11</version.sqlserver.driver>
```

Comment:
jre11 seems to indicate that would not run on Java 1.8? Didn't we mean to keep that for DBZ 2.0?

Reply:
@gunnarmorling Changed, I forgot that we are on Java 11 for building but Java 8 for runtime.